index.html

```diff
@@ -712,101 +712,79 @@ <h2 class='informative'>Vary</h2>
       </section>
     </section>
 
-    <section id="messaging">
-      <h2>Messaging</h2>
+    <section id="notifications">
+      <h2>Notifications</h2>
       <section class="informative">
         <h3>Introduction</h3>
         <p>
-          This section defines when messages are emitted by a Fedora
-          implementation, the minimal set of data contained in those messages and how
-          those data are serialized. These messages may occur synchronously or asynchronously
-          with the API operations that cause them to be emitted. They are typically used to
-          support external integrations. The structure of these messages draws upon the
+          This section defines when notifications are made available by a Fedora
+          implementation, the minimal set of data contained in these notifications and how
+          the data are serialized. Notifications may be emitted synchronously or asynchronously
+          with the API operations that cause them to be emitted. These notifications are typically
+          used to support external integrations. The structure of these notifications draws upon the
           existing [[activitystreams-core]] and [[ldn]] specifications.
-          An implementation is free to choose from any messaging technology so long as
-          the messages conform to what is described in the following sections.
-        </p>
-      </section>
-
-      <section id="emitting-messages">
-        <h3>Emitting Messages</h3>
-        <p>
-          Any operation that causes the state of a resource to durably change MUST cause a
-          message to be emitted with a corresponding resource identifier.
-        </p>
-        <p>
-          A failed operation (HTTP 4xx or 5xx) MUST NOT emit any messages.
-        </p>
-        <p>
-          Messages MUST NOT be emitted until after any changes have been persisted to durable storage.
+          An implementation is free to choose from any transport technology so long as
+          the notifications conform to what is described in the following sections.
         </p>
         <p>
-          Repository start, stop and configuration change operations MAY cause a message to be emitted.
+          Implementers should be aware that some operations cause multiple resources to change.
+          In these cases, there will be a corresponding notification describing each of the changes.
```

Comment:
I'm sympathetic to @awoods, it's difficult to track the requirement deltas in a text diff (but @acoburn's PR is more legible IMO, and that's the cost) . Here's the removals that I see going over it trying to pull all the norm indicators out:
* "Repository start, stop and configuration change operations MAY cause a noun to be emitted."
* "nouns MAY contain a timestamp marking the time the resource was added/modified/deleted."
* "A failed operation (HTTP 4xx or 5xx) MUST NOT emit any nouns."
* "nouns MUST NOT be emitted until after any changes have been persisted to durable storage."
* Emitted events MUST contain **exactly one value** (cardinality removed) for resource identifier/IRI
* Emitted events MUST contain exactly one value for event identifier

I don't think there are any new requirements in this PR, tho some requirements individually explicated have been collapsed. @acoburn does that line up with your intentions? I think the removal of the event identifier is the only one that surprises me, but because it's in a consolidated set of requirements it may have been an oversight? I can't remember if that was discussed.